src/utils/rest-api.js

```diff
@@ -50,17 +50,35 @@ export function connectNotificationsWsUpdateConfig() {
     return reconnectingWebSocket;
 }
 
-function backendFetch(url, init) {
+function handleResponse(response, isJson) {
```

Comment:
I'd prefer 'expectsJson' over 'isJson', everywhere the returned value is expected to match a json structure.